content/en/blog/2024/otel-collector-survey/index.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Insights From The OpenTelemetry Collector Survey
+linkTitle: OpenTelemetry Collector Survey
+date: 2024-05-01
+author: '[Hope Oluwalolope](https://github.com/iamebonyhope) (Microsoft)'
+cSpell:ignore: Oluwalolope distros Servicegraph Pprof
+---
+
+The [OpenTelemetry (OTel) Collector](https://opentelemetry.io/docs/collector/) has become a cornerstone tool for observing and monitoring modern software applications. Recently, the End User SIG conducted a survey to gather feedback from users about their experiences with the OTel Collector. We are excited to share valuable insights based on the 186 responses we received from the community, including details about users’ deployment practices and implementation challenges, to help drive its future direction.
+
+## Key Takeaways
+- Companies typically have medium to large collector deployments:
+  - \>5 Collectors: 125/186
+  - \>10 Collectors: 100/186
+- Building custom binaries/distributions of the Collector is more popular than expected (61/186), with most people using the [OTel Collector Builder](https://github.com/open-telemetry/opentelemetry-collector-builder) to do so (49/61)
+- The vast majority deploy the Collector on Kubernetes (150/186)
+- There's more desire for stability (59), self-observability (53), and config management (59), than for new components (14)
+
+## Detailed Insights
+
+### Deployment Scale and Environment
+Our findings indicate robust use of the OTel Collector at scale, with 53.8%(100/186) of respondents deploying more than 10 collectors, and  22%(41/186) of respondents running between 2 and 5 collectors
+
+![Chart showing how many otel collectors people run in their organization](deployment-scale.png)
+
+Kubernetes is the leading platform for Collector deployment (80.6%), followed by virtual machines (33.3%) and bare metal (10.8%).
+
+![Chart showing where people deploy their otel collectors](deployment-environment.png)
+
+### Usage Scenarios 
+The OTel Collector is primarily used as a gateway (64.5%), demonstrating its pivotal role in aggregating telemetry data from various sources. Daemonsets (51.6%) and sidecars (23.7%) are also popular deployment models, showcasing the flexibility of the OTel Collector in different operational contexts.
+
+![Chart showing the usage scenarios of the otel collector](usage-scenarios.png)
```

Comment:
Some of the charts, we created from the raw data, while some of the charts were taken from the google form summary.
This is because the google form had a way of smartly aggregately the data which was complex when we tried to do it ourselves.

Reply:
I generated all the charts from the raw data now.